reference/constraints/File.rst

```diff
@@ -242,6 +242,17 @@ of given mime types (if an array).
 
 You can find a list of existing mime types on the `IANA website`_.
 
+.. note::
+
+    When using this constraint on a :doc:`FileType field </reference/forms/types/file>`,
+    the value of this option is also used in the ``accept`` attribute of the
```

Comment:
I think this still does not really describe what one needs to do. Just to make it a bit more clear what I mean: The following piece of code would not have the attribute being guessed:

```php
$builder->add('image', FileType::class);
```

However, this would have it:

```php
$builder->add('image');
```

But, in the latter example there is no direct connection to the `FileType` anymore.

Reply:
Your description looks very clear to me ... but it 100% looks like a bug:

These should work the same and guess the accept:

```php
$builder->add('image', FileType::class);
$builder->add('image');
```

These should NOT guess the accept:

```php
$builder->add('image', FileType::class, ['attr' => ['accept' => null]);
$builder->add('image', FileType::class, ['attr' => ['accept' => '...']);
$builder->add('image', FileType::class, ['attr' => ['accept' => ['...', '...']]);
$builder->add('image', null, ['attr' => ['accept' => null]);
$builder->add('image', null, ['attr' => ['accept' => '...']);
$builder->add('image', null, ['attr' => ['accept' => ['...', '...']]);
```

What am I missing here?

Reply:
This is not how the type guessing works. It only guesses anything if the type is not given. I don't think there is anything we can do about that right now. One idea could be to introduce a new option which one could use to opt-in the option guessing even if type is already configured. There is also this pending feature request: symfony/symfony#7868

Reply:
Now I know what you mean! Thanks for the review. I've reworded the description.